### ISSUE TITLE
GPL License violation

### ISSUE THREAD

**Issue by graingert** [mitodl/micromasters#1008]
Currently you use wowjs it's under the GPL. I maintain a fork (wow.js) see https://wowjs.uk

This version is completely UMD compatible and doesn't need any hacks.

You could also use AOS instead

**Comment by pdpinch** [comment]
Thanks for the tip, but I don't think this repo uses wow.js. Were you looking at the current website at http://micromasters.mit.edu/ ? It looks like it does use wow.js.

**Comment by graingert** [comment]
https://github.com/mitodl/micromasters/blob/master/package.json#L84 I'm looking at master

**Comment by pdpinch** [comment]
Ah, github search... how useless can you be?

**Comment by graingert** [comment]
Oh sorry, I didn't use github search :( is there wow.js used instead elsewhere?

**Comment by pdpinch** [comment]
No, not at all. My comment was about the uselessness of github's search tool. 

Again, thanks for the tip. We'll take a look into switching the library.

**Comment by graingert** [comment]
Ah, I misunderstood

**Comment by noisecapella** [comment]
@pdpinch AFAIK wow.js just came with the home page and program page code from the designer. I'm not sure we actually use it at all, and we'll be getting rid of it soon in any case with the new redesign

**Comment by graingert** [comment]
probably worth just nixing it completely from the homepage

**Comment by noisecapella** [comment]
I think so. Thanks for bringing this to our attention!

**Comment by graingert** [comment]
@noisecapella you guys probably want to use something like https://github.com/davglass/license-checker to check for MIT incompatible licenses
